components/layout/navbar/profile-menu/profile-item.tsx

```diff
@@ -0,0 +1,62 @@
+"use client";
```

Comment:
Could be a server component, don't put everything as client for simplicity when not required be careful :)

Reply:
yes i need it, i use useSession to get session's data in the file, this is only available in client side

Reply:
https://nextjs.org/docs/app/building-your-application/optimizing/lazy-loading

Lazy load instead, you just need to change the way this is imported, and maybe the way we use it, but it would be good to avoid DOM rendering